### ISSUE TITLE
What version of Django are you using ?

### ISSUE THREAD

**Issue by waleedelsafty** [Sleepingwell/DjangoRpyDemo#8]
the patterns class was  deprecated  in Django 2.x  and completely removed  in Django 3.x
as well as  render_to_response class

**Comment by Sleepingwell** [comment]
To be honest, I can't remember, but it would have been what was resonably current in about 2012. This was a simple demo I knocked up to demonstrate how one could use R from Django and have not looked at it in 6 years.

I have not used either Django or Rpy in years and am not sure of the status of either of them. Is the idea of using R from Django still interesting?

**Comment by waleedelsafty** [comment]
yes  it is very interesting , specially when you have your app written in Django and you want to do some analysis and create statistical reports for the data  of your app
